README.md

```diff
@@ -216,7 +216,38 @@ Now you have a dedicated server setup for NT;RE. To run it, you will need to be
 ./srcds_run +sv_lan 0 -insecure -console -game neo +ip <YOUR_IP> -maxplayers <1-32> +map <MAP_NAME>
 ```
 #### Dedicated Server on Windows
-TODO (if you're running a dedicated server setup on a windows server (NOT a PC), please help make this part)
+These instructions were tested on Windows Server 2016 and Windows 11 machines, they will probably work in all Windows versions.
+1. Install SteamCMD following these instructions: [LINK](https://developer.valvesoftware.com/wiki/SteamCMD#Windows)
+2. Choose a location for your server to be installed into, for example, `C:\NeotokyoServer\`, and create it. (In my case SteamCMD was also in this location)
+3. Run SteamCMD: `steamcmd.exe`
+4. Enter the following commands in SteamCMD (Note that you need to use an absolute path for the install dir):
+    ```
+    force_install_dir <YOUR_LOCATION>\ognt\
+    login anonymous
+    app_update 313600 validate
+    (wait for it to install)
+    quit
+    ```
+5. Run SteamCMD again, and enter these commands:
+    ```
+    force_install_dir <YOUR_LOCATION>\ntrebuild\
+    (this will be the main directory of your server)
+    login anonymous
+    app_update 244310 validate
+    (wait for it to install)
+    quit
+    ```
+6.  Copy all files from the latest release of NT;RE under `mp\game\neo\` into `<YOUR_LOCATION>\ognt\NeotokyoSource` and replace all existing files.
```

Comment:
NT;RE has mounting through `-neopath`, it shouldn't be overriding OG:NT's files

Reply:
Alright